Rooms Hero Sample/Controllers/RoomServiceClientController.cs

```diff
@@ -140,8 +141,15 @@ public async Task<IActionResult> CreateRoom(string userId)
                 CommunicationUserIdentifier identifier = new CommunicationUserIdentifier(userId);
                 RoomParticipant participant1 = new RoomParticipant(identifier);
                 List<RoomParticipant> invitedParticipants = new List<RoomParticipant> { participant1 };
+                CreateRoomOptions createOption = new CreateRoomOptions()
+                {
+                    ValidFrom = validFrom,
+                    ValidUntil = validUntil,
+                    Participants = invitedParticipants,
+                    PstnDialOutEnabled = pstnDialOutEnabled,
+                };
```

Comment:
The object of options should be inside the constructor params right?

Reply:
in this case is not, this is an internal web API wrapper to call into Rooms SDK. For the purpose of Hero sample, we are using default parameter values (validFrom, validUntil, participant, etc.). Hence we do not need to expose them.